docs/grid-limit-numeric-filter-value.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Limit value in a numeric filter
```

Comment:
All words in the Title have to start with a capital Letter ` Limit Value in a Numeric Filter`